packages/runtime/src/widgets/Search.tsx

```diff
@@ -57,34 +58,45 @@ export const Search: React.FC<SearchProps> = ({
   };
 
   return (
-    <AutoComplete
-      allowClear
-      onSearch={handleSearch}
-      // TODO: Handle on search result select
-      // eslint-disable-next-line @typescript-eslint/no-empty-function
-      onSelect={() => {}}
-      options={options}
-      popupMatchSelectWidth={styles?.width}
-      size="large"
-    >
-      <Input
-        placeholder={placeholder}
-        prefix={<SearchOutlined />}
-        style={{
-          width: styles?.width,
-          height: styles?.height,
-          margin: styles?.margin,
-          borderRadius: styles?.borderRadius,
-          borderWidth: styles?.borderWidth,
-          borderStyle: styles?.borderStyle,
-          borderColor: styles?.borderColor
-            ? getColor(styles.borderColor)
-            : undefined,
-          boxShadow: "none",
-          backgroundColor: styles?.backgroundColor,
-        }}
-      />
-    </AutoComplete>
+    <div>
+      <AutoComplete
+        id={id}
+        allowClear
+        onSearch={handleSearch}
+        // TODO: Handle on search result select
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onSelect={() => {}}
+        options={options}
+        popupMatchSelectWidth={styles?.width}
+        size="large"
+      >
+        <Input
+          placeholder={placeholder}
+          prefix={<SearchOutlined />}
+          style={{
+            width: styles?.width,
+            height: styles?.height,
+            margin: styles?.margin,
+            borderRadius: styles?.borderRadius,
+            borderWidth: styles?.borderWidth,
+            borderStyle: styles?.borderStyle,
+            borderColor: styles?.borderColor
+              ? getColor(styles.borderColor)
+              : undefined,
+            backgroundColor: styles?.backgroundColor,
+            boxShadow: "none",
+          }}
+        />
+      </AutoComplete>
+      <style>
+        {`
+			/* Linear loader animation */
+			#${id ?? ""} {
```

Comment:
Check for id block and just don't include style block if it's empty. Otherwise this is just extra elements.